docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+You can install add-ons on a Kubernetes cluster using `kubectl apply -f`.
+
+This page lists some of the available add-ons.
+
+## Networking
+
+* [Weave Net](https://github.com/weaveworks/weave-kube) is a fast, reliable pod network that carries on working even in the face of network partitions, and doesn't have any infrastructure or database dependencies. Install it on a cluster whose kubelets are configured to expect a [CNI network plugin](/docs/admin/network-plugins/):
```

Comment:
+1

I think we do need a 1 sentence description of each add-on though otherwise users have to click all the links to know what they're looking at. Keeping it to 1 sentence and linking to more verbose docs at the appropriate URL totally makes sense though.